adam-core/src/main/scala/org/bdgenomics/adam/converters/FastqRecordConverter.scala

```diff
@@ -143,10 +145,25 @@ class FastqRecordConverter extends Serializable with Logging {
     // get fields for first read in pair
     val readName = trimTrailingReadNumber(lines(0).drop(1))
     val readSequence = lines(1)
-    val readQualities = lines(3)
+
+    if (stringency == ValidationStringency.STRICT && lines(3) == "*" && readSequence.length > 1)
+      throw new Exception(s"Fastq quality must be defined")
+    else if (stringency == ValidationStringency.STRICT && lines(3).length != readSequence.length)
+      throw new Exception(s"Fastq sequence and quality strings must have the same length")
+
+    val readQualities =
```

Comment:
Emit warnings but keep going if possible and fix invalid input data are slightly different things.  I think with a parameter or parameters presented to the user I'd support this change.